frontend/plugins/mongolian_ui/src/modules/ebarimt/settings/stage-in-return-ebarimt-config/components/ReturnEBarimtConfigTable.tsx

```
@@ -147,13 +147,13 @@ const returnEbarimtColumns: ColumnDef<IReturnEbarimtConfigRow>[] = [
     accessorKey: 'hasVat',
     header: () => {
       const { t } = useTranslation('mongolian');
-      return <RecordTable.InlineHead label={t('has-vat')} icon={IconToggleLeft} />;
+      return <RecordTable.InlineHead label={t('has-vat', 'Has Vat')} icon={IconToggleLeft} />;
     },
     cell: ({ cell }) => {
       const { t } = useTranslation('mongolian');
```

ESLint

[error] React Hook "useTranslation" is called in function "cell" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)